requirements.txt

```diff
@@ -11,7 +11,7 @@ ipywidgets>=7.5.1  # package
 jinja2>=2.10  # package
 jsonpatch>=1.22  # package
 jsonschema>=2.5.1  # package
-mistune>=0.8.4  # package
+mistune>=0.8.4,<2.0.0  # package
```

Comment:
I was unable to figure out how to un-pin `pip` and resolve `aiobotocore` dependencies. The pinned version of `pip` needs to know about this constraint.